Packs/CommonScripts/Scripts/HttpV2/HttpV2_test.py

```diff
@@ -0,0 +1,60 @@
+"""Base Script for Cortex XSOAR - Unit Tests file
+
+Pytest Unit Tests: all funcion names must start with "test_"
+
+More details: https://xsoar.pan.dev/docs/integrations/unit-testing
+
+MAKE SURE YOU REVIEW/REPLACE ALL THE COMMENTS MARKED AS "TODO"
+
+"""
+
+import json
```

Comment:
I feel there's more to test here, we have almost no unit tests

Reply:
I tested most usecases in the TPB